markdown.go

```diff
@@ -58,18 +61,21 @@ const (
 	TOC                                            // Generate a table of contents
 	OmitContents                                   // Skip the main contents (for a standalone table of contents)
 
-	CommonHtmlFlags HTMLFlags = UseXHTML
+	CommonHTMLFlags HTMLFlags = UseXHTML
 
 	CommonExtensions Extensions = NoIntraEmphasis | Tables | FencedCode |
 		Autolink | Strikethrough | SpaceHeaders | HeaderIDs |
 		BackslashLineBreak | DefinitionLists | Smartypants |
 		SmartypantsFractions | SmartypantsDashes | SmartypantsLatexDashes
 )
 
+// DefaultOptions is a convenience variable with all the options that are
+// enabled by default.
 var DefaultOptions = Options{
 	Extensions: CommonExtensions,
 }
 
+// TODO: this should probably be unexported. Or moved to node.go
```

Comment:
Future. It needs a bit of investigation to figure out whether it's a private field or should be a part of LinkData at node.go.